samples/ORKCatalog/ORKCatalog/ResultTableViewProviders.swift

```diff
@@ -470,6 +473,62 @@ class TappingIntervalResultTableViewProvider: ResultTableViewProvider {
     }
 }
 
+/// Table view provider specific to an `ORKToneAudiometryResult` instance.
+class ToneAudiometryResultTableViewProvider: ResultTableViewProvider {
```

Comment:
I appreciate having this here in ORKCatalog, but we also need this case added to ORKTest. Localization testing uses the ORKTest app only.